parachains/runtimes/starters/seedling/src/lib.rs

```diff
@@ -65,7 +65,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_version: 9270,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 1,
+	transaction_version: 7,
```

Comment:
```suggestion
	transaction_version: 1,
```
Do not bump `transaction_version`